### DISCUSSION TITLE
White List Issue

### DISCUSSION THREAD

**Discussion by DanMarquesPT** [Romanitho/Winget-AutoUpdate#305] [General]
Hi all,

Appreciating all the help. Have setup my Intune to Deploy Winget-AutoUpdate and using Intune to deploy the Policy. So far so good apart from the Devices not getting the apps listed on the White list, even if the log identifies the use of White List.

19:04:54 - Running in System context
19:04:54 - Activated WAU GPO Management detected, comparing...
19:04:54 - No Changed settings detected
19:04:54 - Running Post Update actions:
19:04:54 - -> Checking prerequisites...
19:04:54 - -> Prerequisites checked. OK
19:04:54 - -> Checking if Winget is installed/up to date
19:04:55 - -> WinGet is Installed/up to date
19:04:56 - -> Winget sources reseted.
19:04:56 - -> ListPath setting was missing. Fixed with empty string.
19:04:56 - -> ModsPath setting was missing. Fixed with empty string.
19:04:56 - -> Checking Mods Directory:
19:04:56 - -> The mods directory was already secured!
19:04:56 - Post Update actions finished
19:04:56 - Notification Level: Full. Notification Language: English
19:04:56 - Checking internet connection...
19:04:56 - Connected !
19:04:58 - Winget Version: v1.4.10173
19:04:58 - WAU current version: 1.17.1
19:04:58 - WAU AutoUpdate is Enabled.
19:04:58 - WAU AutoUpdate Pre-release versions is Enabled
19:04:59 - WAU is up to date.
19:05:00 - WAU uses White List config
19:05:00 - Checking application updates on Winget Repository...
-> Available update : Google Chrome. Current version : 111.0.5563.65. Available version : 111.0.5563.111.
-> Available update : Teams Machine-Wide Installer. Current version : 1.2.0.34161. Available version : 1.6.00.4472.
19:05:02 - Google Chrome : Skipped upgrade because it is not in the included app list
19:05:02 - Teams Machine-Wide Installer : Skipped upgrade because it is not in the included app list
19:05:02 - No new update.

Thanks in advance for the help

**Comment by KnifMelti** [comment]
![image](https://user-images.githubusercontent.com/102996177/227694083-a9fdde7e-a438-4af2-ba2c-9bc50caa87fe.png)

**Comment by DanMarquesPT** [comment]
It is active. Log assumes I am using white list but does not follow the list

![image](https://user-images.githubusercontent.com/128723180/227706842-d9377967-6856-4caf-9287-b0051ed68c3a.png)
![image](https://user-images.githubusercontent.com/128723180/227706872-023ac5e9-47d1-44af-947f-198d428177f2.png)

**Reply by KnifMelti** [reply]
And:
![image](https://user-images.githubusercontent.com/102996177/227708234-c13f2b93-d00a-41a8-a04a-3d90a3ee1b56.png)

**Reply by KnifMelti** [reply]
Your log does not include WAU uses External Lists from: GPO

**Reply by DanMarquesPT** [reply]
I will activate this and try it next week. Thanks for your patience.
